wcivf/apps/people/templates/people/includes/_person_about_card.html

```diff
@@ -0,0 +1,40 @@
+{% if object.wikipedia_bio or object.associatedcompany_set.exists or object.cv %}
+<section class="ds-card">
+  <div class="ds-card-body">
+    <h2 class="ds-candidate-name ds-h3">
+      About {{ object.name }}
+    </h2>
+    <p>
+      {% if object.cv %}
```

Comment:
I think this `if` should be before the opening `<p>`? Otherwise if there is no CV you will have an empty `<p>` element

Reply:
Or perhaps that `<p>` just isn't needed?